.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
> if you weren't using the .styleci.yml

Yes, but I want to use the config file. It is simpler for contribution, most of CI services use this method.

If this method will be abandoned and deleted, so we will not be able to use it.